packages/open-payments/src/client/outgoing-payment.ts

```diff
@@ -39,11 +58,35 @@ export const getOutgoingPayment = async (
   validateOpenApiResponse: ResponseValidator<OutgoingPayment>
 ) => {
   const { axiosInstance, logger } = deps
-  const { url } = args
+  const { url, accessToken } = args
 
   const outgoingPayment = await get(
     { axiosInstance, logger },
-    args,
+    { url, accessToken },
+    validateOpenApiResponse
+  )
+
+  try {
+    return validateOutgoingPayment(outgoingPayment)
+  } catch (error) {
+    const errorMessage = 'Could not validate outgoing payment'
+    logger.error({ url, validateError: error?.message }, errorMessage)
+
+    throw new Error(errorMessage)
+  }
+}
+
+export const createOutgoingPayment = async (
```

Comment:
Here the only other validation I could think of doing on a newly created outgoing payment (that wasn't already covered in `validateOutgoingPayment`) was checking whether `sentAmount.value === 0`. Feels like a weird behaviour to already start processing a payment before returning it, but does the caller care?

Reply:
I agree that does somehow seem less concerning than a non-zero `receivedAmount` on a new incoming payment.